lingua-franca-ref.txt

```diff
@@ -1 +1 @@
-master
+run-zephyr-tests
```

Comment:
Is this correct?

Reply:
Thanks, that has been merged so I change back to master